docs/source/contributing/writing-docs.adoc

```diff
@@ -159,3 +160,11 @@ $ rake build
 ----
 
 If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+[[building-openshift-docs-automatically]]
+=== Building the OpenShift Documentation Automatically
+
+To automatically get the view of {project} documentation integrated into the OpenShift documentation, open the link:https://ci.centos.org/job/minishift-docs/[minishift-docs] job in your browser and trigger it by clicking on `Build with Parameters` and specifying your repo URL in `REPO` and branch name containing your documentation changes in `BRANCH` parameters respectively.
```

Comment:
"To stage a version of the documentation based on a given repository and branch, you can use the CentOS CI  link:https://ci.centos.org/job/minishift-docs/[minishift-docs] job.

Reply:
I don't see a "Build with Parameters". I would think to do so, one needs to be logged in, but from what I recall from our discussions, we are not supposed to log into the instance. I cannot find the username/password either anyway. We have the access token, but that's a different beast, right? Something is not adding up here.

Reply:
> I would think to do so, one needs to be logged in, but from what I recall from our discussions, we are not supposed to log into the instance. I cannot find the username/password either anyway. We have the access token, but that's a different beast, right? Something is not adding up here.

Yes, one need to log into Jenkins UI. `username/password` is mentioned in cdk mojo secret page.

> Something is not adding up here.

The piece which is missing here is user need to log into Jenkins UI to able to see the build with parameters option. Not sure how to tackle this in the documentation. 
Does following seems reasonable?

```
Currently, only maintainers have rights to trigger this job. If you want to stage your own version of the documentation, reach out to `#minishift` channel on https://freenode.net/[Freenode IRC] or send a request mail on https://lists.minishift.io/admin/lists/minishift.lists.minishift.io[mailing list].
```
 WDYT?

Reply:
@hferentschik 

It is looking like

![screenshot from 2017-11-29 15-11-17](https://user-images.githubusercontent.com/1132451/33368441-b70c6970-d517-11e7-97be-022da41617a2.png)

Reply:
So why don't we not use URL based triggering as with the release. 
My issues is not that contributors cannot do it. I don't think we need to do anything special for them. My problem is that I can see the access token for triggering via URL documented in our internal docs, but not username/password.